tsinfer/formats.py

```diff
@@ -1197,8 +1210,29 @@ def sites_time(self, value):
 
     @property
     def sites_alleles(self):
+        """
+        The "raw" alleles list for each site, as passed in when adding sites. Note that
+        if an ancestral allele is defined, it need not be the zeroth element. The
+        allele list for a site may therefore differ in order from that returned by the
```

Comment:
The last sentence is incorrect isn't it? Also it should be the variants() method we're talking about, not sites I think?

Reply:
Yes, sorry - still on the final pass to check through docs and add extra tests